voxblox_ros/src/tsdf_server.cc

```diff
@@ -52,6 +52,30 @@ void TsdfServer::getServerConfigFromRosParam(
   } else {  // Default case is gray.
     color_mode_ = ColorMode::kGray;
   }
+
+  // Color map for intensity pointclouds.
+  std::string intensity_colormap("rainbow");
+  float intensity_max_value = 100.0;
```

Comment:
this default value is a little buried, could it be set from a constexpr that is in the header?